controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -48,21 +53,37 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
   }
 
   private def uploadFile(projectId: ProjectId)(implicit accessToken: AccessTokenContent): Route = post {
-    entity(as[ProjectUpdateFileRequest]) { request =>
-      onComplete {
-        for {
-          validateResponse <- wdlService.validateFile(request.projectFile.content)
-          uploadResponse <- wdlService.uploadFile(projectId, request.projectFile, request.version, accessToken.userId)
-        } yield (validateResponse, uploadResponse) match {
-          case (Right(_), Right(_)) => Right(StatusCodes.OK)
-          case (Left(_), Right(_))  => Right(StatusCodes.Created)
-          case (_, Left(response))  => Left(response.getMessage)
+    formFields('version.as[PipelineVersion](stringToPipelineVersion).optional, 'path.as[Path].optional) {
```

Comment:
I think it is a problem from the test case. But if mentors approve it I`ll aprrove it to.